app-config.yaml

```
@@ -275,6 +275,10 @@ pagerduty:
 lighthouse:
   baseUrl: ${LIGHTHOUSE_BASEURL}
 
+# Plugin: Permission
+permission:
+  enabled: true
```

```suggestion
enabled:
  permissions: ${PERMISSIONS_ENABLED}
```

Please keep this consistent with other plugins inside the `enabled` block.

---

It is set like this as it is required by the [Permission Plugin](https://backstage.io/docs/permissions/getting-started#2-enable-and-test-the-permissions-system) within Backstage. Now, do we still want to include the change below? 
```yaml
enabled:
  permissions: ${PERMISSIONS_ENABLED}
```
I am worried that it could cause some slight confusion as they are very similiar and users might think setting one is enough. It also increases the number of environment variables that need to be managed.

---

Yes some plugins and Backstage will have the enabled attribute in a specific section. It also more traditional. We will have to change our existing config.